test/test-bopomofo.c

```diff
@@ -1690,6 +1691,49 @@ void test_KB()
     test_KB_MPS2();
 }
 
+void test_chewing_phone_to_bopomofo()
+{
+    char *u8phone;
+    char *rt;
+    int  len;
+    uint16_t phone;
+    uint16_t expect;
+
+    start_testcase(NULL, fd);
+
+    u8phone = "\xE3\x84\x86\xE3\x84\xA3" /* ㄆㄣ */ ;
+    phone = UintFromPhone(u8phone);
+    expect = 1104;
```

Comment:
Hi @jserv , 
I copy this test case from `test_uint_and_phone()`  function in test/test-key2phone.c .
https://github.com/chewing/libchewing/blob/699578a5ea361621ff4601afbae2aa694be74978/test/test-key2pho.c#l24

and I want to know what does the "more flexible ways to generate expected results" mean? 
does it mean the more simple case or other ?

thanks!